config/nginx/conf.d/cantusdb-development.conf

```diff
@@ -0,0 +1,80 @@
+# This file is configured for deployment of the CantusDB project in local development.
+# When building the project locally, replace the contents of cantusdb.conf with the
+# contents of this file.
+
+# server {
+#     listen 80;
+# 
+#     server_tokens off;
+# 
+#     location ^~ /.well-known/acme-challenge/ {
+#         root /var/www/lego;
+#     }
+# 
+#     location / {
+#         return 301 https://$host$request_uri;
+#     }
+# }
+ 
+# server {
+#     # Redirect all https traffic for mass.cantusdatabase.org to cantusdatabase.org
+#     listen 443 ssl;
+# 
+#     server_name mass.cantusdatabase.org;
+# 
+#     ssl_certificate /etc/nginx/ssl/live/certificates/cantusdatabase.org.crt;
+#     ssl_certificate_key /etc/nginx/ssl/live/certificates/cantusdatabase.org.key;
+# 
+#     location / {
+#         return 301 https://cantusdatabase.org$request_uri;
+#     }
+# }
+ 
+server {
```

Comment:
Looking at this now I'm realizing we don't have a `server_name` in this block?

Reply:
I noticed the other day that we aren't doing host checking at the Django level either, but I thought it was ok because we were doing it here, but it looks like we aren't. We should do both: https://docs.djangoproject.com/en/5.0/topics/security/#host-headers-virtual-hosting

Reply:
If I understand, this is less relevant to of this file in particular, and more to the project in general - correct?

I think this is beyond the scope of this PR. Let me open a couple of new issues. Thanks for catching this!

Reply:
I guess it depends if we want to validate the Host with nginx at all. If so, then that will be different for staging and production. I'm not yet seeing clear answers on the google.